### PR TITLE
LPS-137004 Always use SelectCategory component

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-collection-filter-category/src/main/resources/META-INF/resources/page.jsp [hunk 1]
```diff
@@ -20,30 +20,17 @@
...
 </p>
 
-<c:choose>
-	<c:when test="<%= fragmentCollectionFilterCategoryDisplayContext.isSingleSelection() %>">
-		<clay:dropdown-menu
-			cssClass="bg-light font-weight-bold form-control form-control-select form-control-sm text-left w-100"
-			displayType="secondary"
-			dropdownItems="<%= fragmentCollectionFilterCategoryDisplayContext.getDropdownItems() %>"
-			label="<%= fragmentCollectionFilterCategoryDisplayContext.getSelectedAssetCategoryTitle() %>"
-			title="<%= fragmentCollectionFilterCategoryDisplayContext.getAssetCategoryTreeNodeTitle() %>"
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-collection-filter-category/src/main/resources/META-INF/resources/page.jsp:30]: Are these functions used anywhere else? Can we remove it?

`fragmentCollectionFilterCategoryDisplayContext.getDropdownItems()`
`fragmentCollectionFilterCategoryDisplayContext.getSelectedAssetCategoryTitle()`
`fragmentCollectionFilterCategoryDisplayContext.getAssetCategoryTreeNodeTitle()`

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/5442

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**